core/api/BUCK

```diff
@@ -155,6 +156,16 @@ madge_check(
     srcs = [":src"],
 )
 
+test_unit(
+    name = "unit-tests",
+    srcs = [":src"] + [":test_src"] + glob([".env", "galoy.yaml"]),
+    config_file = "test/unit/jest.config.js",
+    env_file = ".env",
```

Comment:
I would prefer to bring this into the file instead of relying on the on disk state like in the [Tiltfile](https://github.com/GaloyMoney/galoy/blob/595cf23073bd9ca50b85c3f5e28e0ab6da254ef9/dev/Tiltfile#L137).

Ideally we would be able to delete all the .env files and rely on environment defined in code. I don't know if that's feasible yet - we can merge as is to not mess with the existing workflow - but I think that should be the goal once legacy workflows are entirely deprecated.